tests/publiccloud/check_registercloudguest.pm

```diff
@@ -19,7 +19,9 @@ use strict;
 use utils;
 use publiccloud::utils;
 use publiccloud::ssh_interactive 'select_host_console';
-use version_utils 'is_sle';
+
+my $path = is_sle('=15-SP2') ? '/usr/sbin/' : '';    # 15-SP2 is the oldest version that needs fullpaths
```

Comment:
Whyare you removing the check on sles `>15` ?

Reply:
As per the comment, ` # 15-SP2 is the oldest version that needs fullpaths` i.e. No other version needs the fullpath, so no need to use it.